civihr_default_theme/scss/components/_leave-request-modal.scss

```diff
@@ -12,6 +12,10 @@ $row-margin-bottom: 15px;
   overflow: auto;
 }
 
+.chr_leave-request-modal__comments_empty {
+  margin-bottom: 20px;
```

Comment:
At this point we dont have a specific style guide, so everything is mentioned to match the design. Also we have different margin bottom in the same file. 
Its a good point though. Same goes for 2nd review point.